boot/mynewt/mcuboot_config/include/mcuboot_config/mcuboot_config.h

```diff
@@ -85,6 +85,16 @@
 #if MYNEWT_VAL(BOOTUTIL_BOOTSTRAP)
 #define MCUBOOT_BOOTSTRAP 1
 #endif
+#if MYNEWT_VAL_CHOICE(BOOTUTIL_DOWNGRADE_PREVENTION, version)
+#define MCUBOOT_DOWNGRADE_PREVENTION                     1
```

Comment:
Maybe a comment here. It is not clear that `MCUBOOT_DOWNGRADE_PREVENTION` is used in a `#if defined` construct, whereas `MCUBOOT_DOWNGRADE_PREVENTION_SECURITY_COUNTER` is used as a boolean.

Reply:
comment added